### PR TITLE
feat: add ability to configure font for chat application

### LINKED ISSUE #1329: Add ability to choose font for chat application
### Description

We can add this ability to themes or to any other place. We need it to keep our system whitelabel which is quite important.

### Use case/motivation

_No response_

### Related issues

_No response_

### Confidential information

- [X] I confirm that do not share any confidential information

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**
Add ability to configure font for chat application through the  css variable in the theme.

Issues:

- Issue #1329 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

File: apps/chat/src/types/themes.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ export interface Theme {
   displayName: string;
   colors: Record<string, string>;
   'app-logo': string;
+  fontFamily: Record<string, string>;
```
Comment (mikitabut) [inline apps/chat/src/types/themes.ts]: Why font family is record? Give me example please how data will look
I expect just simple string in this font family like font family in CSS

Comment (mikitabut) [inline apps/chat/src/types/themes.ts]: Please don't forget to update documentation about themes customization

**Review by IlyaBondar — DISMISSED** [review]

File: apps/chat/src/pages/api/themes/styles.ts [hunk 2]
```diff
@@ -103,6 +119,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
         wrapCssContents(`.${theme.id}`, [
           generateColorsCssVariables(theme.colors),
           generateUrlsCssVariables({ 'app-logo': theme['app-logo'] }),
+          generateFontCssVariables(theme.fontFamily),
```
Comment (IlyaBondar) [inline apps/chat/src/pages/api/themes/styles.ts]: I think it should work like 
`generateFontCssVariables('theme-font', theme['font-family']`
i.e. just string should be provided in theme

File: apps/chat/src/types/themes.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/types/themes.ts]: Changed to the string.
Thank you for advice!

Comment (denys-kolomiitsev) [reply apps/chat/src/types/themes.ts]: Documentation updated.

File: apps/chat/src/pages/api/themes/styles.ts (hunk 2 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/pages/api/themes/styles.ts]: Changed as suggested.
Function `generateFontCssVariables` changed accordingly

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1332.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — DISMISSED** [review]

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1332.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/pages/_app.tsx [hunk 3]
```diff
@@ -14,10 +14,10 @@ import { HomeProps } from '.';
 import { createStore } from '@/src/store';
 import '@/src/styles/globals.css';
 
-const inter = Inter({
+export const inter = Inter({
```
Comment (mikitabut) [inline apps/chat/src/pages/_app.tsx:17]: do not export this inter font - create separate const for it and use it where it needed
Import something from FE to server code is bad practise

File: apps/chat/src/types/themes.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ export interface Theme {
   displayName: string;
   colors: Record<string, string>;
   'app-logo': string;
+  'font-family': string;
```
Comment (mikitabut) [inline apps/chat/src/types/themes.ts]: set this as optional

Comment (denys-kolomiitsev) [reply apps/chat/src/types/themes.ts]: Changed to be optional.

File: apps/chat/src/pages/_app.tsx (hunk 3 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/pages/_app.tsx:17]: Otherwise it doesn't work.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1332.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**